app/app.go

```diff
@@ -587,9 +587,7 @@ func (app *TerraApp) Name() string { return app.BaseApp.Name() }
 
 // BeginBlocker application updates every begin block
 func (app *TerraApp) BeginBlocker(ctx sdk.Context, req abci.RequestBeginBlock) abci.ResponseBeginBlock {
-
-	if ctx.ChainID() == core.ColumbusChainID && ctx.BlockHeight() == core.SwapDisableForkHeight {
-		// Make min spread to one to disable swap
+	if ctx.ChainID() == core.ColumbusChainID && ctx.BlockHeight() == core.SwapDisableForkHeight { // Make min spread to one to disable swap
```

Comment:
I would probably move this comment down above the app.MarketKeeper.SetParams method call which is where the actual value is set :)

Reply:
I didn't touch this actually. This is a conflict because I based the changes on v1.0.4 release and the main branch obviously diverged...